crates/sui-core/src/consensus_handler.rs

```diff
@@ -143,8 +143,16 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
             &self.metrics,
         );
 
+        self.metrics
+            .consensus_committed_subdags
+            .with_label_values(&[&consensus_output.sub_dag.leader.header.author.to_string()])
```

Comment:
just a note, the `author` will change as part of this https://github.com/MystenLabs/sui/pull/9576  - we'll just need to do the key conversion before this point

Reply:
Also cardinality for this will be `100` , not the end of the world, but I would say let's make sure we don't do this everywhere